python.html.markdown

```diff
@@ -123,8 +123,11 @@ not False  # => True
 # A string can be treated like a list of characters
 "This is a string"[0]  # => 'T'
 
-# % can be used to format strings, like this:
-"%s can be %s" % ("strings", "interpolated")
+#String formatting with %
+#Even though the % string operator will be deprecated on Python 3.1 and removed later at some time, it may still be #good to know how it works.
```

Comment:
Odd formatting on this line. Could you reformat to 80-char width, and comment correctly?